src/api/python/cvc5.pxi

```diff
@@ -4051,6 +4104,7 @@ cdef class Term:
             - **Int    :** Returns a Python int
             - **Real   :** Returns a Python Fraction
             - **BV     :** Returns a Python int (treats BV as unsigned)
+            - **FF     :** Returns a Python int (gives the FF integer representative of smallest absolute value)
```

Comment:
I think this should say string?

Reply:
Right now, the implementation does return an integer. E.g., for the finite field of order 5 is can return -2, -1, 0, 1, or 2.

Is there a reason it should return a string?